### DISCUSSION TITLE
[RESOLVED] Specify Username and Password on Mobile

### DISCUSSION THREAD

**Discussion by samir1** [Vinzent03/obsidian-git#304] [Problems] [answered]
I'm trying to specify my username and password when cloning a remote repo but there doesn't really seem like a place to enter it when typing in the remote url. Are there text boxes to enter the username and password when cloning? I've tried to set my username and password in the settings but it doesn't seem to fix the issue.
![IMG_D17627CA4A88-1](https://user-images.githubusercontent.com/6413003/188993526-a2ced93d-fa80-417b-9025-5feb0d373160.jpeg)

**Comment by Vinzent03** [comment, marked as answer]
You currently need to set you pr username and password/personal access token in the settings and clone afterwards. What exactly happens on cloning?

**Reply by samir1** [reply]
Thanks! Got it working!
